{
  "type": "class",
  "name": "NbtAccounterException",
  "qualified_name": "net.minecraft.nbt.NbtAccounterException",
  "package": "net.minecraft.nbt",
  "modifiers": [
    "public",
    "class"
  ],
  "annotations": [],
  "description": "",
  "inheritance": {
    "extends": null,
    "implements": [],
    "inheritance_tree": [
      "java.lang.Object",
      "java.lang.Throwable",
      "java.lang.Exception",
      "java.lang.RuntimeException",
      "net.minecraft.nbt.NbtException"
    ]
  },
  "nested_classes": [],
  "fields": [],
  "constructors": [
    {
      "name": "NbtAccounterException",
      "modifiers": [
        "public"
      ],
      "parameters": [
        {
          "name": "p_301729_",
          "type": "String",
          "annotations": []
        }
      ],
      "description": "",
      "deprecated": false
    }
  ],
  "methods": [],
  "enum_constants": [],
  "since": null,
  "deprecated": false,
  "source_file": "Frameworks/1.20.4/net/minecraft/nbt/NbtAccounterException.html"
}